### DISCUSSION TITLE
Can you remove one of the delays?

### DISCUSSION THREAD

**Discussion by WHHwxiaop** [zompi2/UE4EnhancedCodeFlow#37] [Q&A] [answered]
Can  remove one of the delays?

**Comment by zompi2** [comment]
Hi!
To remove one of the running Delays action you can use `FFlow::StopAction(GetWorld(), Handle);` where Handle is the `FECFHandle` returned by the `FFlow::Delay` function.
Is this it what you're asking for?

**Reply by WHHwxiaop** [reply]
Use FFlow::StopAction(GetWorld(), Handle)，how can I get this paused delay to start delaying again from the beginning?

**Reply by zompi2** [reply, marked as answer]
`FFlow::StopAction` does not pause the delay, but it terminates it. In order to start delaying again from the beginning you need to run `FFlow::Delay` again.

To pause and resume the action use `FFlow::PauseAction(GetWorld(), Handle)` and `FFlow::ResumeAction(GetWorld(), Handle)`

However there is no way to reset the running or paused action. This actually might be a good thing to add.

**Reply by WHHwxiaop** [reply]
Got it. Thanks.
